end-to-end-tests/build.gradle

```diff
@@ -27,6 +27,8 @@ dependencies {
 test.enabled = false
 
 task endToEndTest(type: Test) {
+  inputs.property "integration.date", LocalTime.now() // so it runs at every invocation
```

Comment:
This seems like a hack. Was this just for local testing?

Reply:
this is the RIGHT way to make sure that you can re-run a given gradle task. We use that for AT in Besu too.